### PR TITLE
Browser tests with cypress

### PR DESCRIPTION
**Pull request by gr2m**
follow up / closes #8

### CODE REVIEW

**Comment by gr2m** [comment]
ZOMG it's all green on first try I love Cypress @bahmutov

I’ll move config back into the config file as it’s required anyway. Could you have a quick glance if anything else looks good?

**Comment by bahmutov** [comment]
I will take a look tomorrow - did you push into my fork or is it in yours ?

**Comment by gr2m** [comment]
this is mine, I wanted to keep your PR intact for reference

**Comment by gr2m** [comment]
@bahmutov would still appreciate a quick glance reivew, but i'll move forward with this

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "chai": "^4.1.2",
     "compression-webpack-plugin": "^1.1.6",
     "coveralls": "^3.0.0",
+    "cypress": "^1.4.2",
```
Comment (bahmutov) [inline package.json:53]: can you check if `1.4.2 ` is working? I ran it two days ago and seems there was some breaking change between `1.4.1` and `1.4.2` which breaks `nock` :(

Comment (gr2m) [reply package.json:53]: Yeah it seems to work: https://travis-ci.org/gr2m/octokit-rest-browser-experimental/jobs/338642214#L504

I don't use nock in the tests, I start a separate server process that I send the requests to, maybe that's why we don't run into the bug?

**Pull request merged**